akka-cluster/src/main/scala/akka/cluster/VectorClock.scala

```diff
@@ -187,5 +187,11 @@ final case class VectorClock(
     VectorClock(mergedVersions)
   }
 
+  def prune(removedNode: Node): VectorClock =
```

Comment:
This removes a "column" for a given node? (i.e. the version vector gets shorter)